source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -40,45 +42,71 @@ guide, consider importing the following classes in your program:
    import static com.mongodb.client.model.mql.MqlValues.*;
    import static java.util.Arrays.asList;
 
-The examples use the ``Arrays.asList()`` method to create a list of
-aggregation operations that can be passed as a pipeline to the
-``aggregate()`` method.
-
-To construct an operation, first select a method that takes a value of
-type ``TExpression`` as a parameter, such as ``Projections.computed()``
-or ``Filters.expr()``. To reference document fields in an aggregation
-expression, obtain the "current" document being processed
-in the pipeline stage by calling the ``current()`` method. 
+To access document fields in an aggregation expression, you need to reference the
+document being processed by the aggregation pipeline. Use the
+``current()`` method to refer to this document.
 
-Since the aggregation operations are typed, you must specify the type of each
-field you reference using a typed variation of the ``getField()``
+To access the value of a field, you must use the appropriately typed
 method, such as ``getString()`` or ``getDate()``. The following code
-shows how to reference an integer field called ``quantity``:
+shows how to reference a string field called ``name``:
 
 .. code-block:: java
    :copyable: true
+   
+   current().getString("name")
 
-   var myField = current().getInteger("quantity");
-
-To reference a constant value, use the ``of()`` method, which converts
-primitive types to types that can be used in aggregation operations. The
-following code shows how to reference a constant of value ``1.0``:
+To specify a value in an operation, pass it to the ``of()`` method to
+convert it to a valid type. The following code shows how to reference a
+value of ``1.0``:
 
 .. code-block:: java
    :copyable: true
+   
+   of(1.0)
 
-   var myNum = of(1.0);
+To create an operation, chain a method to your field or value reference.
+You can build more complex operations by chaining additional methods.
 
-When you finish composing an aggregation operation, you can pass it to
-an aggregation builder method, such as ``match()`` or ``project()`` to
-complete an aggregation pipeline stage. To learn more about these methods,
-see :ref:`Aggregates Builders <aggregates-builders>`.
+The following example creates an operation to find patients in New
+Mexico who have visited the doctor’s office at least once. The operation
+performs the following actions:
 
-.. note::
-   
-   While these new operations can be used within existing aggregation
-   methods that take ``TExpression`` values, methods that return
-   ``Bson`` values are not usable with this feature.
+- Checks if the size of the ``visitDates`` array is greater than ``0``
+  by using the ``gt()`` method 
+- Checks if the ``state`` field value is “New Mexico” by using the
+  ``eq()`` method
+
+The ``and()`` method links these operations so that the pipeline stage
+matches only documents that meet both criteria.
```

Comment:
[suggestion]
Consider combining this above in the bulleted list. Something like

"Combines these two operations by using the `and()` method". 

If you do that, you might need to slightly reword the introductory sentence though.

Reply:
I think the way it is now is a bit easier to follow, in the other case the last bullet point would reference the previous two, which I think is more challenging. For me, it's easier to understand this conceptually as two separate operations that are then linked